packages/components/psammead-podcast-promo/README.md

```diff
@@ -53,7 +53,7 @@ This component is a basic `p` that is styled to be used for the synopsis/descrip
 
 ### Components: `PodcastPromo.Card.CallToAction`
 
-This component intended to be used as the "call to action" within each card. The child text is positioned alongside an icon. Any props passed to this component are forwarded to the `span` element that wraps the text. This component is `aria-hidden` to screen readers.
+This component intended to be used as the "call to action" within each card. The child text is positioned alongside an icon. Any props passed to this component are forwarded to the `p` element that wraps the text.
```

Comment:
Is this referring to the 'Episodes' text? Not sure that's a CTA?

Reply:
Yea - I think this is another example of where the spec changed after we wrote the first issue, but the issue wasn't updated. At some point I think this was intended to be more of a CTA. I'll change the naming of this component.